src/Configuration/Provisioning/ProvisioningForm/PolicyForm/ProvisioningFormSourceCustomCatalogHeader.jsx

```diff
@@ -23,7 +23,7 @@ const ProvisioningFormSourceCustomCatalogHeader = () => {
         <ActionRow.Spacer />
         <Hyperlink
           target="_blank"
-          destination={`${LMS_BASE_URL}${CUSTOMER_CATALOG_PATH(formData?.enterpriseUUID || null)}`}
+          destination={`${LMS_BASE_URL}${lmsCustomerCatalog.queryBy(formData?.enterpriseUUID || null)}`}
```

Comment:
Is it important that null be passed rather than undefined?
```suggestion
          destination={`${LMS_BASE_URL}${lmsCustomerCatalog.queryBy(formData?.enterpriseUUID)}`}
```